arangod/RestHandler/RestReplicationHandler.cpp

```diff
@@ -2554,16 +2556,29 @@ void RestReplicationHandler::handleCommandHoldReadLockCollection() {
     return;
   }
 
-  if (!body.isObject()) {
+  VPackSlice collection, ttlSlice, idSlice;
+  std::string rebootId;
+
+  if (body.isObject() && body.hasKey("collection") && body.hasKey("ttl") && body.hasKey("id")) {
```

Comment:
How about using `StaticStrings::*`?

Reply:
not all of above are available and i think then one should do that effort concerted. i tend to do this for classes, which i write from scratch

Reply:
Can we please use StaticStrings for this?!